packages/js/tests/dashboard/components/site-kit-setup-widget.test.js

```diff
@@ -0,0 +1,56 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { SiteKitSetupWidget } from "../../../src/dashboard/components/site-kit-setup-widget";
+
+describe( "SiteKitSetupWidget", () => {
+	const defaultProps = {
+		installUrl: "https://example.com/install",
+		activateUrl: "https://example.com/activate",
+		setupUrl: "https://example.com/isSetup",
+		isConnected: false,
+		isActive: false,
+		isSetupCompleted: false,
+		isInstalled: false,
+		onRemove: jest.fn(),
+		onRemovePermanently: jest.fn(),
+		learnMoreLink: "https://example.com/learn-more",
+	};
+
+	it( "renders the widget with install button", () => {
+		render( <SiteKitSetupWidget { ...defaultProps } /> );
+		expect( screen.getByText( "Install Site Kit by Google" ) ).toBeInTheDocument();
```

Comment:
You don't check if this is a button, even though the `it` text mentions that. Same for below.